app/lib/employment_income_subtotals.rb

```diff
@@ -53,12 +53,8 @@ def entitles_child_care_allowance?
     employments_excluding_self_employments.any?(&:entitles_childcare_allowance?)
   end
 
-  def employment_names
-    [@employment_result].compact.flat_map(&:employments).map(&:employment_name)
-  end
-
-  def employments_payments
-    [@employment_result].compact.flat_map(&:employments).map(&:employment_payments)
+  def payment_based_employments
```

Comment:
@starswan What do you think of using "employments" as a method name?

Reply:
I thought what I have used is more descriptive. There are 3 'employments' concepts in the domain, so wanted to pick a name that reflected this specific one (the one based on payments)